### DISCUSSION TITLE
Synology Ejabberd "synocommunity" .spk archive?

### DISCUSSION THREAD

**Discussion by ghost** [processone/ejabberd#3796] [Questions] [answered]
Hi!

Recently updated ejabberd on my DSM to ver. 21.07-2 and now the package fails to start on DSM 6.2.4-25556 Update 2.

Is there an archive somewhere that I can DL an older .spk of ejabberd?  The "beta" version was working flawlessly prior to the update, now the package does not start.

Would love to regain ejabberd functionality on my older DSM, but need an archive or some help compiling an older version.

Appreciate any help!

Opi

**Comment by licaon-kter** [comment]
Who packages that exactly? Ask them?

**Comment by badlop** [comment, marked as answer]
It seems you are referring to this package: https://synocommunity.com/package/ejabberd

You can see the changes to build that package here:
https://github.com/SynoCommunity/spksrc/commits/master/cross/ejabberd

And https://synocommunity.com/ says:

> Contribute
> 
> We welcome package related bug reports, patches and feature requests on our [issue tracker](https://github.com/SynoCommunity/spksrc/issues). You can also support us by donating to our contributors.

Looking there, there's an issue titled: [ejabberd: update to v21.07 #4959](https://github.com/SynoCommunity/spksrc/pull/4959)


So, you should report this in that bug tracker.
